examples/seat-adjuster/Dockerfile

```diff
@@ -44,7 +44,7 @@ RUN velocitas init -v
 
 WORKDIR /workspace/app
 
-RUN pyinstaller --clean -F -s -p src src/main.py
+RUN pyinstaller --hiddenimport=pkg_resources.extern --clean -F -s -p src src/main.py
```

Comment:
I will also try to remove the fixed `pyinstaller` version above, introuced in https://github.com/eclipse-velocitas/vehicle-app-python-sdk/pull/85

The  `pyinstaller` shall have been fixed (see https://github.com/pyinstaller/pyinstaller/issues/8554). I will test if just removing the fixed version will solve the problem or if we manually must fetch a newer pyinstaller version.